README.md

```diff
@@ -1 +1,101 @@
-unity-hierarchy-behaviour
+# unity-hierarchy-behaviour
+
+# What is it?
+Its a collection of GameObject extension methods to allow for runtime instantiation of MonoBehaviours that include an Initialize method that takes type-safe arguments.
```

Comment:
Maybe put specifically that they are children in here. It's quite core to the feature